src/components/SubclassCheckbox.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+	<div class="querybuilder__include-subclasses">
+		<input
+			class="querybuilder__include-subclasses-checkbox"
+			type="checkbox"
+			id="include-subclasses"
+			v-model="checked"
+		/>
+		<label for="include-subclasses">
+			<span
+				v-i18n="{msg: 'query-builder-include-subclasses'}" />
+		</label>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+export default Vue.extend( {
+	name: 'SubclassCheckbox',
+	computed: {
+		checked: {
+			get(): boolean {
+				return this.$store.getters.subclasses;
+			},
+			set( value: boolean ): void {
+				this.$store.dispatch( 'setSubclasses', value );
```

Comment:
No, this is missing the info about the conditionIndex. In general, I think it would be the better idea if this component doesn't talk to the store at all and just has a value prop and emits an input event when the checkbox is toggled. Let's let `QueryCondition` deal with the talking to the store.

Reply:
oh, true.. :sweat: totally forgot about the index.